src/Campaigns/Blocks/shared/components/Pagination/index.tsx

```diff
@@ -0,0 +1,81 @@
+import {__, sprintf} from '@wordpress/i18n';
+import cx from 'classnames';
+import {ChevronLeft, ChevronRight} from './icons';
+
+import './styles.scss';
+
+export default ({currentPage, totalPages, setPage}: PaginationProps) => {
+    if (1 >= totalPages) {
+        return null;
+    }
+
+    const nextPage = currentPage + 1;
+    const previousPage = currentPage - 1;
+
+    return (
+        <div className="give-campaign-components-pagination">
+            <div className="give-campaign-components-pagination__pages">
+                <div className="give-campaign-components-pagination__pages-links">
+                    {previousPage > 0 ? (
+                        <a
+                            href="#"
+                            title={__('Previous page', 'give')}
+                            className="give-campaign-components-pagination__pages-links-arrow"
+                            onClick={(e) => {
+                                e.preventDefault();
+                                setPage(previousPage);
+                            }}
+                        >
+                            <ChevronLeft />
+                        </a>
+                    ) : (
+                        <a className="give-campaign-components-pagination__pages-links-arrow-disabled">
+                            <ChevronLeft />
+                        </a>
+                    )}
+
+                    {[...Array(totalPages)].map((e, i) => {
+                        const page = i + 1;
+                        return (
+                            <a
```

Comment:
These pagination links should probably just be a button element

Reply:
I agree

Reply:
Resolved https://github.com/impress-org/givewp/pull/7708/commits/060f986c79f969560f121ca246cae7450276d82c